Packages/com.chisel.graph/Chisel/ChiselGraph.cs

```diff
@@ -0,0 +1,61 @@
+using Chisel.Core;
+using System.Collections.Generic;
+using UnityEngine;
+using XNode;
+
+namespace Chisel.Nodes
+{
+    [CreateAssetMenu(fileName = "New Chisel Graph", menuName = "Chisel Graph")]
+    public class ChiselGraph : NodeGraph
+    {
+        public ChiselGraphNode active;
```

Comment:
since the active node is set through SetActiveNode, wouldn't it make sense to make this either readonly ( get; private set; } or just private?

Reply:
The user selected active node is basically the root tree node, the graph parse starts there, so it need to be serialized
Public is definitely not right form a API standpoint I guess, maybe [SerializeField]private is better

Reply:
[SerializeField] private would be better imho yeah, since I can imagine a user not understanding the code and try to modify the active node for whatever reason, and then being confused things going crazy